modules/react/combobox/lib/ComboboxMenuList.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import {commonColors} from '@workday/canvas-kit-react/tokens';
+import {
+  createElemPropsHook,
+  createSubcomponent,
+  ExtractProps,
+} from '@workday/canvas-kit-react/common';
+import {useMenuModel} from '@workday/canvas-kit-react/menu';
+import {ListBox, ListProps} from '@workday/canvas-kit-react/collection';
+
+import {useComboboxModel} from './useComboboxModel';
+
+export interface ComboboxMenuListProps<T = any>
+  extends Omit<ExtractProps<typeof ListBox>, 'children'> {
+  /**
+   * Children of the Combobox. Should contain a `<Combobox.Target>`, a `<Combobox.Content>`
+   */
+  children: ListProps<T>['children'];
```

Comment:
we use this type quite frequently: `React.ReactNode | ((item: T) => React.ReactNode)` or `ListProps['children']`. I wonder if it's worth adding that to our lower level components, or use `ListProps['children']` where ever it expects a list of items like action bar list, or tabs. Sometimes we define the type, sometimes we use listProps

Reply:
I've been converting to use `ListProps['children']` instead. I was considering having these interfaces extend from `ListProps`. The problem is we tend to add custom JSDocs to the `children` prop. The actual value might only matter for doc parsing.